### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by Priya-Patel31**
Please review my avatar component !!

### CODE REVIEW

**Comment by sks-21** [comment]
The code looks nice, I really liked the use of selector lists for making the code modular. Good work!

**Comment by harshitadaswani** [comment]
The naming conventions you have used are really nice!

File: component/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<h1 class="styling-color">Avatar</h1>
```
Comment (Ankur9669) [inline component/avatar/avatar.html:1]: Don't use H1 for displaying avatar as it would affect SEO.

File: component/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<p class="sub-content">
+  Avatars are images that users can set as their profile picture. On GitHub,
+  they're always going to be rounded squares. They can be custom photos,
+  uploaded by users, or generated as Identicons as a placeholder.
+</p>
+<h2>Example of Image Avater</h2>
```
Comment (Ankur9669) [inline component/avatar/avatar.html:7]: Avatar spelling is incorrect on line 7

**Review by Ankur9669 — APPROVED** [review]
All looks fine to me go ahead and merge it.

**Pull request merged**